src/core/ComplexFilter/index.tsx

```diff
@@ -164,9 +161,8 @@ export default function ComplexFilter<
     }
 
     const newValue =
-      (value as Value<DefaultMenuSelectOption, true>)?.filter(
-        (item) => item !== option
-      ) || null;
+      (value as DefaultMenuSelectOption[])?.filter((item) => item !== option) ||
```

Comment:
change `Value<DefaultMenuSelectOption, true>` to `DefaultMenuSelectOption[]` to be more readable